pkg/provider/aws/aws_test.go

```diff
@@ -0,0 +1,5 @@
+package aws
+
+import "github.com/klothoplatform/klotho/pkg/core/coretesting"
+
+type DagExpectation = coretesting.ResourcesExpectation
```

Comment:
What's the benefit of this type alias? (and similarly in `resources_test.go`)

Reply:
Just makes it shorter. In the context of the provider, "DAG" is the resource DAG, but without that context (ie, in coretesting) it needs to be specified.
Easy to change if we don't like it.

Reply:
It adds a bit of indirection for little gain, so imo it's not worth it — but definitely not a blocker.